modinfo.lua

```diff
@@ -1,12 +1,12 @@
 local modinfo = {
-	name			= 'Spring: 1944',
-	shortName		= 'S44',
-	version			= '$VERSION',
-	game			= 'Spring: 1944',
-	shortGame		= 'S44',
-	mutator			= 'Official',
-	description		= 'Epic World War II RTS!',
-	url				= 'http://www.spring1944.net/',
+	name			= 'Steam Spring 1944',
+	shortName		= 'ss44',
+	version			= 'alpha-$VERSION',
+	game			= 'Steam Spring 1944',
+	shortGame		= 'ss44',
+	mutator			= 'notAlab distribution',
+	description		= 'Epic World War II RTS',
+	url				= 'http://notalab.com/products/ss44/',

```

Comment:
The URL is spring1944.net not this URL... :/// And the Mutator needs to be correct too. And it's not Steam Version, and No, it's not ss44, and no, it's not an alpha-*version. ;) Need to be correct !

Reply:
Those complains are invalid, because this file changes won't merge anywhere. 

This is one of the manual changes which I have to do during the merge to avoid propagation of alpha setup to master branch. In previous merge this was not present

Reply:
Btw check this file is in conflicted state in global overview (+check what is last commit in master branch). Both because of this.

Reply:
Well, that's a PR... If something shall not be merged, then you should not include it in the PR (by means of a branch, right?)

Reply:
Maybe I'm noob in using the github UI. I always handle `modinfo.lua` manually while propagating changes from master > alpha > beta > release branch, because each branch have different game name in given branch for lobby distinction purposes. 

In ideal world alpha branch wont be needed after this merge, so from now only master > beta > release will stay for purpose of Steam release. We will see, until its clear I will keep current modinfo.lua in alpha branch in current state. 

As I said once you give me green for finishing the PR, i will manage this file manually anyway.

Reply:
I prefer to do an usual PR leaded merge process... But don't worry... Just go ahead with the other topics

Reply:
Ok, all topics I could solved (except potential removing of 2 old widgets - but those are not blockers for merge for me) are solved.

Waiting for your 2 TODOs. What would help me is setting some scope of definition of done of those TODOs. Maybe I can help with them or at least I will have idea how much time it will take (so I can focus on different things).

Reply:
We have the license issue...

Reply:
addressed in [03b32ce] - more in other comments